c/netdb/netdb.go

```diff
@@ -0,0 +1,29 @@
+package netdb
+
+import (
+	"github.com/goplus/llgo/c/socket"
+	_ "unsafe"
+
+	"github.com/goplus/llgo/c"
+)
+
+const (
+	LLGoPackage = "decl"
+)
+
+type AddrInfo struct {
+	AiFlags     c.Int
+	AiFamily    c.Int
+	AiSockType  c.Int
+	AiProtocol  c.Int
+	AiAddrLen   c.Uint
+	AiCanOnName *c.Char
+	AiAddr      *socket.SockAddr
+	AiNext      *AddrInfo
+}
+
+//go:linkname GetAddrInfo C.getaddrinfo
+func GetAddrInfo(host *c.Char, port *c.Char, addrInfo *AddrInfo, result **AddrInfo) c.Int
```

Comment:
The style of these functions needs to be changed. `GetAddrInfo` should be changed to `Getaddrinfo` to maintain consistency with the auto-generation tool, which does not expect to process word breaks in the original symbol (e.g., getaddrinfo) into camel case naming.